evm_body.yaml

```diff
@@ -1602,29 +1602,29 @@ transaction_object:
     from:
       type: string
       description: 20 Bytes - The address the transaction is sent from.
-      pattern: '^0[xX][0-9a-fA-F]+$'
+      pattern: '^0[xX][0-9a-fA-F]{40}$'
     to:
       type: string
       description: 20 Bytes - The address the transaction is directed to
       default: '0xd46e8dd67c5d32be8058bb8eb970870f07244567'
-      pattern: '^0[xX][0-9a-fA-F]+$'
+      pattern: '^0[xX][0-9a-fA-F]{40}$'
     gas:
       type: string
       description: Integer of the gas provided for the transaction execution.
-      pattern: '^0[xX][0-9a-fA-F]+$'
+      pattern: '^0[xX]([1-9a-fA-F]+[0-9a-fA-F]*|0)$'
```

Comment:
It doesn't matter much, but you can leave out the `+` which maybe makes the regex a little easier to follow.